exercism/go
# Implement yacht for go

Create an implementation of yacht for the go track.

* Readme, tests and example solution
* metadata and gen.go in .meta directory
* Change config.json to refer to yacht, using self-generated uuid. 
* Change the "poker" configuration to depend on yacht ("I can see this exercise being a precursor to poker since it has similar concepts without requiring the parsing of the cards." -- petertseng)
